### PR TITLE
Added labels for a11y.

### PR DESCRIPTION
**Pull request by dlabrecq**
## Description

Added labels for a11y. Note that the labels are positioned off-screen, hidden from view, but still accessible to screen readers.

The approach is documented here: http://webaim.org/techniques/css/invisiblecontent/

This addresses Gabriel's code review comments from:

https://github.com/patternfly/patternfly/pull/500
## Changes

Write a list of changes the PR introduces
- Added labels for a11y
- Added style to position labels off-screen
- Change 03
## Link to rawgit and/or image

https://rawgit.com/dlabrecq/patternfly/table-view-dist/dist/tests/datatables.html

This is an image:

screen shot 2016-10-11 at 4 22 49 pm
## PR checklist (if relevant)
- [x] **Cross browser**: works in IE9
- [ ] **Cross browser**: works in IE10
- [x] **Cross browser**: works in IE11
- [ ] **Cross browser**: works in Edge
- [x] **Cross browser**: works in Chrome
- [x] **Cross browser**: works in Firefox
- [x] **Cross browser**: works in Safari
- [ ] **Cross browser**: works in Opera
- [x] **Responsive**: works in extra small, small, medium and large view ports.
- [x] **Preview the PR**: An image or a URL for designer to preview this PR is provided.

### CODE REVIEW

**Comment by dlabrecq** [comment]
@cardosogabriel Please review. This should address your code review comments from #500.

File: src/less/datatables.less [hunk 1]
```diff
@@ -349,6 +349,15 @@
 // Selection column
 .datatable-pf-select {
   width: 33px;
+  // Hidden labels for a11y
+  label {
```
Comment (andresgalante) [inline src/less/datatables.less]: @dlabrecq Bootstrap has a class `sr-only` to hide labels.
You can read more about it here http://getbootstrap.com/css/#forms-inline

File: tests/pages/_includes/widgets/datatables/tmpl/datatable.html [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <table class="table table-striped table-bordered table-hover" id="table">
   <thead>
     <tr>
-      <th><input type="checkbox" name="selectAll"></th>
+      <th><label>Select all rows</label><input type="checkbox" name="selectAll"></th>
```
Comment (andresgalante) [inline tests/pages/_includes/widgets/datatables/tmpl/datatable.html]: so, here we can just add the `sr-only` class 😄

**Comment by andresgalante** [comment]
@dlabrecq I love that you've started pushing a11y. Thanks a lot!

File: tests/pages/_includes/widgets/datatables/tmpl/datatable.html (hunk 2 above)
Comment (dlabrecq) [reply tests/pages/_includes/widgets/datatables/tmpl/datatable.html]: Thanks, Andres! That's exactly what I needed.

PR has been updated.

**Comment by andresgalante** [comment]
👍 excellent thanks

**Pull request merged**